src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf

```diff
@@ -310,6 +310,30 @@
                 <source>This value does not match the expected {{ charset }} charset.</source>
                 <target>Šī vērtība neatbilst sagaidāmajai rakstzīmju kopai {{ charset }}.</target>
             </trans-unit>
+            <trans-unit id="81">
+                <source>This is not a valid Business Identifier Code (BIC).</source>
+                <target>Šī vērtība nav derīgs Biznesa Identifikācijas Kods (BIC).</target>
+            </trans-unit>
+            <trans-unit id="82">
+                <source>Error</source>
+                <target>Kļūda</target>
+            </trans-unit>
+            <trans-unit id="83">
+                <source>This is not a valid UUID.</source>
+                <target>Šis nav derīgs UUID.</target>
+            </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>Šai vērtībai jābūt vairākas reizes atkārtotai {{ compared_value }}.</target>
```

Comment:
Probably a false positive, but Google Translate says that `atkārtotai` should be `atkārtotam`  Could you please check it? Thanks.

Reply:
well, it depends if the noun is in male or female form. in this case "vērtība" is female, which means it should have "atkārtotai" which is female form instead of "atkārtotam" which is male

Reply:
Thanks for the explanation! So it was a false positive. Let's ignore it.